### PR TITLE
Adds compat data for AnimationEvent

### LINKED ISSUE #965: Add compat data for AnimationEvent
MDN pages to migrate:
- [ ] [Web/API/AnimationEvent](https://developer.mozilla.org/en-US/docs/Web/API/AnimationEvent)
- [ ] [Web/API/AnimationEvent/AnimationEvent](https://developer.mozilla.org/en-US/docs/Web/API/AnimationEvent/AnimationEvent)
- [ ] [Web/API/AnimationEvent/animationName](https://developer.mozilla.org/en-US/docs/Web/API/AnimationEvent/animationName)
- [ ] [Web/API/AnimationEvent/elapsedTime](https://developer.mozilla.org/en-US/docs/Web/API/AnimationEvent/elapsedTime)
- [ ] [Web/API/AnimationEvent/initAnimationEvent](https://developer.mozilla.org/en-US/docs/Web/API/AnimationEvent/initAnimationEvent)
- [ ] [Web/API/AnimationEvent/pseudoElement](https://developer.mozilla.org/en-US/docs/Web/API/AnimationEvent/pseudoElement)

The data should be migrated into the file `api/AnimationEvent.json`.

For more information on how to help with this work, see https://developer.mozilla.org/en-US/docs/MDN/Contribute/Structures/Compatibility_tables

### PR DESCRIPTION
**Pull request by bennypowers**
Fixes #965

### CODE REVIEW

**Comment by bennypowers** [comment]
see https://github.com/mdn/browser-compat-data/issues/965

File: api/AnimationEvent.json [hunk 1]
```diff
@@ -0,0 +1,232 @@
...
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null,
+            "prefix": "webkit"
+          },
+          "chrome": [
+            {
+              "version_added": "1",
```
Comment (Elchi3) [inline api/AnimationEvent.json]: Please swap this around, the more relevant version is the unprefixed one. (the first element will be rendered as the main compat cell in the MDN tables, other version ranges will be put into the expandable details.)

```json
        "chrome": [
            {
              "version_added": "43"
            }
            {
              "version_added": "1",
              "prefix": "webkit"
            },
          ],
```

File: api/AnimationEvent.json [hunk 2]
```diff
@@ -0,0 +1,280 @@
+{
+  "api": {
+    "AnimationEvent": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent",
+        "support": {
+          "webview_android": {
+            "version_added": null,
```
Comment (jpmedley) [inline api/AnimationEvent.json]: Change to `true`.

File: api/AnimationEvent.json [hunk 3]
```diff
@@ -0,0 +1,280 @@
...
+            "prefix": "webkit"
```
Comment (jpmedley) [inline api/AnimationEvent.json]: Unprefixed in 43.

File: api/AnimationEvent.json [hunk 4]
```diff
@@ -0,0 +1,280 @@
...
+          },
+          "chrome": [
+            {
+              "version_added": "43"
```
Comment (jpmedley) [inline api/AnimationEvent.json]: Is this how we note when something was unprefixed? @Elchi3

File: api/AnimationEvent.json [hunk 5]
```diff
@@ -0,0 +1,280 @@
...
+            },
+            {
+              "version_added": "1",
+              "prefix": "webkit"
+            }
+          ],
+          "chrome_android": {
+            "version_added": "43"
```
Comment (jpmedley) [inline api/AnimationEvent.json]: `true` for original version added. `"43"` is for when the prefix was removed.

File: api/AnimationEvent.json [hunk 6]
```diff
@@ -0,0 +1,280 @@
...
+          "standard_track": true,
+          "deprecated": false
+        }
+      },
+      "AnimationEvent": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent/AnimationEvent",
+          "support": {
+            "webview_android": {
+              "version_added": false
```
Comment (jpmedley) [inline api/AnimationEvent.json]: Should be `true` and prefix removed in `"43"`. Repeat for Chrome and Chrome Android.

File: api/AnimationEvent.json [hunk 7]
```diff
@@ -0,0 +1,280 @@
...
+          "mdn_url": "https://developer.mozilla.org/docs/Web/API/AnimationEvent/animationName",
+          "support": {
+            "webview_android": {
+              "version_added": null
+            },
+            "chrome": {
+              "version_added": true
+            },
+            "chrome_android": {
+              "version_added": "43"
```
Comment (jpmedley) [inline api/AnimationEvent.json]: All three versions of Chrome should have the same data as for the constructor.

File: api/AnimationEvent.json [hunk 8]
```diff
@@ -0,0 +1,280 @@
...
+              "version_added": "10"
+            },
+            "opera": {
+              "version_added": "10"
+            },
+            "safari": {
+              "version_added": "4"
+            },
+            "webview_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/AnimationEvent.json]: Should be `true`.

File: api/AnimationEvent.json [hunk 9]
```diff
@@ -0,0 +1,280 @@
...
+            },
+            "edge_mobile": {
+              "version_added": null
+            },
+            "firefox_android": {
+              "version_added": "6",
+              "version_removed": "23"
+            },
+            "opera_android": {
+              "version_added": null
```
Comment (jpmedley) [inline api/AnimationEvent.json]: `true`.

File: api/AnimationEvent.json (hunk 4 above)
Comment (Elchi3) [reply api/AnimationEvent.json]: Yes, the array indicates support ranges. 
So, there is a support range starting from version 43 where this is fully available and unprefixed.
Then there is a second support ranges where it was available (version not known) and prefixed. 
The rendered tables take the first array element as the one to render in the main cell. The other support ranges will be treated as secondary and put into the expandable info. See e.g. https://developer.mozilla.org/en-US/docs/Web/API/CanvasRenderingContext2D/imageSmoothingEnabled#Browser_compatibility

**Comment by Elchi3** [comment]
Well done! We're almost there.
I think there is one more member of this interface that I've missed in my last review: https://developer.mozilla.org/en-US/docs/Web/API/AnimationEvent/elapsedTime

**Pull request merged**